test/svg-test-helper.js

```diff
@@ -110,60 +110,54 @@ export const getPathCommandsFromContainer = (container) => {
   return parseSvgPathCommands(commandStr);
 };
 
-const exhibitsShapeSequence = (commandString, shapeSeqeuence) => {
+export const exhibitsShapeSequence = (commandString, shapeSeqeuence) => {
   const commands = parseSvgPathCommands(commandString);
   return commands.every(
     (command, index) => command.name === shapeSeqeuence[index]
   );
 };
 
-export const svgExpectations = {
```

Comment:
Just getting rid of this entirely so utilize can just be used in assertions within test files.